src/components/Market/MarketFilter/index.tsx

```diff
@@ -0,0 +1,313 @@
+
+type Range = {
+  min: number;
+  max: number;
+};
+
+type RangeOption = {
+  limit: Range;
+  label: string;
+};
+
+const coreOccupancyOptions: RangeOption[] = [
+  { limit: { min: 0, max: 1 }, label: 'Show All' },
+  { limit: { min: 0, max: 0.25 }, label: '0% - 25%' },
+  { limit: { min: 0.25, max: 0.5 }, label: '25% - 50%' },
+  { limit: { min: 0.5, max: 0.75 }, label: '50% - 75%' },
+  { limit: { min: 0.75, max: 1 }, label: '75% - 100%' },
+];
+
+type DurationOption = {
+  duration?: Timeslice;
+  label: string;
+};
+
+const durationOptions: DurationOption[] = [
+  { duration: undefined, label: 'Show All' },
+  { duration: WEEK_IN_TIMESLICES, label: '1 week' },
+  { duration: 2 * WEEK_IN_TIMESLICES, label: '2 weeks' },
+  { duration: 3 * WEEK_IN_TIMESLICES, label: '3 weeks' },
+  { duration: 4 * WEEK_IN_TIMESLICES, label: '4 weeks' },
+];
+
+interface MarketFilterProps {
+  onChange: (_: MarketFilterOptions) => void;
+}
+
+type FilterState = {
+  startDate: Date | null;
+  endDate: Date | null;
+  occupancyIndex: number;
+  durationIndex: number;
+  priceRange: [number, number] | undefined;
+};
+
+const defaultFilterState: FilterState = {
+  startDate: null,
+  endDate: null,
+  occupancyIndex: 0,
+  durationIndex: 0,
+  priceRange: [0, 100],
+};
+
+interface FilterItemProps {
+  label: string;
+  children: React.ReactNode;
+}
+
+const FilterItem = ({ label, children }: FilterItemProps) => {
+  const theme = useTheme();
+  return (
+    <Stack direction='column' gap='0.5rem'>
+      <Typography
+        sx={{
+          color: theme.palette.common.black,
+          fontWeight: 600,
+          fontSize: '0.75rem',
+        }}
+      >
+        {label}
+      </Typography>
+      <FormControl fullWidth>{children}</FormControl>
+    </Stack>
+  );
+};
+
+export const MarketFilter = ({ onChange }: MarketFilterProps) => {
+  const theme = useTheme();
+  const {
+    state: { symbol, decimals },
+  } = useRegionXApi();
+
+  // Filters
+  const [startDate, setStartDate] = useState<Date | null>(null);
+  const [endDate, setEndDate] = useState<Date | null>(null);
+  const [occupancyIndex, setOccupancyIndex] = useState(0);
+  const [durationIndex, setDurationIndex] = useState(0);
+  const [priceRange, setPriceRange] = useState<[number, number] | undefined>([
+    0, 100,
+  ]);
+
+  const [filter, setFilter] = useState<FilterState>(defaultFilterState);
+
+  // Filter popover
+  const [anchorEl, setAnchorEl] = useState<HTMLButtonElement | null>(null);
+  const open = Boolean(anchorEl);
+  const id = open ? 'popover-filters' : undefined;
+  const openFilters = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const closeFilters = () => {
+    setAnchorEl(null);
+
+    setStartDate(filter.startDate);
+    setEndDate(filter.endDate);
+    setOccupancyIndex(filter.occupancyIndex);
+    setDurationIndex(filter.durationIndex);
+    setPriceRange(filter.priceRange);
+  };
+
+  const clearFilters = () => {
```

Comment:
When clearing all filters, we should also reset the price range from 0 to the maximum value.